### DISCUSSION TITLE
Can anyone help me reduce this?

### DISCUSSION THREAD

**Discussion by sanjay11102** [vercel/next.js#31809] [Help]
I was working on a project and i want to improve the performance of the current website but was not able to reduce this. can anyone help me with this issue?

<img width="958" alt="Screenshot 2021-11-26 at 1 09 09 AM" src="https://user-images.githubusercontent.com/23434566/143494037-f70623b0-25e6-4b09-8c3d-442812e7ba33.png">

**Comment by sanjaiyan-dev** [comment]
Hi, Please check this [answer, I also had same confusion :)](https://github.com/vercel/next.js/discussions/31204#discussioncomment-1613577)
